features/cart/shopping_cart/change_currency_of_cart.feature

```diff
@@ -0,0 +1,21 @@
+@shopping_cart
+Feature: All of my cart's values get updated to the currency of my choosing
+    In order to know how much I have to pay in my currency
+    As a Visitor
+    I want to see every cash amount in my chosen currency
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And that channel allows to shop using the "EUR" currency
+        And that channel also allows to shop using the "GBP" currency with exchange rate 2.00
+        And that channel uses the "EUR" currency by default
+        And the store has a product "The Pug Mug" priced at "€10.00"
```

Comment:
![the pug mug](https://cloud.githubusercontent.com/assets/6213903/17962774/a436de64-6ab2-11e6-8b25-ddbd0f3f6bd4.png)